### PR TITLE
Appdata strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**
# Description
More in line with the changes made to other stings.
What is "<content_attribute id="sex-homosexuality">none</content_attribute>" good for?
There are already a bunch of other categories to cover it…?

Fixes #(issue)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce.
- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: data/com.usebottles.bottles.appdata.xml.in [hunk 1]
```diff
@@ -4,32 +4,32 @@
...
         <ul>
-          <li>Create your bottles using preconfigured environments or create your own</li>
-          <li>Run every executable (.exe/.msi) in your bottles, directly from the context menu form your file manager</li>
+          <li>Create bottles using preconfigured environments or create your own</li>
+          <li>Run executables (.exe/.msi) in your bottles, directly from the context menu of your file-manager</li>
           <li>Automated detection of applications installed in your bottles</li>
-          <li>Add environment variables easily</li>
-          <li>Override DLLs directly from bottle preferences</li>
+          <li>Add environment variables quickly</li>
+          <li>Override DLLs directly from per-bottle preferences</li>
```
Comment (comradekingu) [inline data/com.usebottles.bottles.appdata.xml.in:22]: or?

File: data/com.usebottles.bottles.appdata.xml.in [hunk 2]
```diff
@@ -240,7 +240,7 @@
               <p>Happy 2021🎉!</p>
               <p>News:</p>
               <ul>
-                <li>The new experimental Bottle Versioning feature is here! Save the status of a bottle and restore it later with a click.</li>
+                <li>The new experimental bottle Versioning feature is here! Save the status of a bottle and restore it later with a click.</li>
```
Comment (comradekingu) [inline data/com.usebottles.bottles.appdata.xml.in:243]:
```suggestion
                <li>The new experimental bottle versioning feature is here! Save the status of a bottle and restore it later with a click.</li>
```

**Comment by mirkobrombin** [comment]
That line come with a bundle of content rating. Thanks for your contribution, I will do the review as soon as possible.

File: data/com.usebottles.bottles.appdata.xml.in (hunk 1 above)
Comment (mirkobrombin) [reply data/com.usebottles.bottles.appdata.xml.in:22]: It sounds better

**Pull request merged**